src/util/amount.rs

```diff
@@ -14,12 +14,7 @@
 //! We refer to the documentation on the types for more information.
 //!
 
-use std::default;
-use std::error;
-use std::fmt::{self, Write};
-use std::ops;
-use std::str::FromStr;
-use std::cmp::Ordering;
+use core::{ops, str::FromStr, fmt::{self, Write}, default, cmp::Ordering};
```

Comment:
I'm personally not a great fan of nesting, but I don't have a policy on it.

Reply:
yeah, that's a bit much, splitting up